### PR TITLE
Feat/offering heading

### PR DESCRIPTION
**Pull request by Wandkleister**

### CODE REVIEW

File: _includes/js/offering-header.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+		this.bindEvents();
+	}
+
+	bindEvents() {
+		document.addEventListener("scroll", throttle(this.scrollHandler.bind(this), 150));
+	}
+
+	updateTextPathOffset(offsetTop, offsetBottom) {
+		//this.textPathTop.setAttribute("style", `transform: translate3d(${offsetTop}px, 0, 0)`);
+		//this.textPathBottom.setAttribute("style", `transform: translate3d(${offsetBottom}px, 0, 0)`);
```
Comment (tam-kis) [inline _includes/js/offering-header.js:22]: comments left

**Comment by luismeyer** [comment]
i think on mobile screens the overflow should be hidden

File: _includes/offering.liquid [hunk 2]
```diff
@@ -1,3 +1,54 @@
...
+        </ul>
+      </div>
+      <div class="col-3">
+        <h3 class="offering__title">Marketing Growth</h3>
+        <ul class="offering__list">
+          <li class="offering__item">Visual & Editorial Content</li>
+          <li class="offering__item">Marketing & Personalization</li>
+          <li class="offering__item">Performance & Growth</li>
+          <li class="offering__item">Analytics</li>
+        </ul>
```
Comment (luismeyer) [inline _includes/offering.liquid:47]: couldn't we use a json file with data and map it into the template. Thats what liquid is for afterall right? 😅

Comment (tam-kis) [reply _includes/offering.liquid:47]: Of course, we did it like this because we don't have time. I David manages to find some time we will adapt it.

Comment (timbru31) [reply _includes/offering.liquid:47]: Should we track this as an issue here on GH?

Comment (Wandkleister) [reply _includes/offering.liquid:47]: I'm already on it.

Comment (Wandkleister) [reply _includes/offering.liquid]: Adjusted ✅

**Review by luismeyer — APPROVED** [review]

**Pull request merged**